src/Files.App/Filesystem/StorageItems/ShellStorageFile.cs

```diff
@@ -85,10 +85,17 @@ public static IAsyncOperation<BaseStorageFile> FromPathAsync(string path)
 			return Task.FromResult<BaseStorageFile>(null).AsAsyncOperation();
 		}
 
-		private static ShellFileItem GetFile(string path)
+		private static ShellFileItem? GetFile(string path)
```

Comment:
Why would this be null?

Reply:
The default value of ShellFileItem is null, which could be now raised in the `catch`.